### ISSUE TITLE
Autoform modals work only once

### ISSUE THREAD

**Issue by ghost** [mozfet/meteor-autoform-materialize-modals#4]
Autoform modals only work once if i destroy or close the modal without click cancel button or click out side the modals, but if destroy or close the modals with click the cancel button, everything works fine....

is this a bug?

##cancel this issue
this is only happen on  playground, but its working on my  server... the only problem now is 

`Uncaught TypeError: Cannot read property 'removeChild' of null
    at Array.complete (materialize_materialize.js?hash=e69bc7be2cfa7671c49705c6482043784d010064:2150)
    at p (materialize_materialize.js?hash=e69bc7be2cfa7671c49705c6482043784d010064:409)
    at c (materialize_materialize.js?hash=e69bc7be2cfa7671c49705c6482043784d010064:397)`

when closing the modal...

**Comment by mozfet** [comment]
I do not have any issues with my dev or prod environment. That being said I still need to upgrade to latest Meteor and Autoform and Materialize; thus currently I have not tested the package with latest versions… so you might be ahead of me. I am planning to do an upgrade this week.

There were some bugs in this area a while ago. It was due to issues in MaterialiseCSS so I fixed it with a few workarounds to make dismissals and the modal overlay work correctly… it might be that the issues have now been addressed in Materialize and my workarounds be interfering...

I also saw the error, but could not locate it, it does not seem to affect functionality though. Will look into it when I upgrade later this week.

**Comment by ghost** [comment]
thanks, btw maybe you can explain more about how to use modals on update the docs, coz i cant get it to work...

**Comment by mozfet** [comment]
do I understand correctly that you are not getting an update form using Collection2 to work? Or are you using it with autoform type method, or normal type (with submit hook)? Could you post some code?

**Comment by ghost** [comment]
"update" problem solved

im waiting for the upgrade..  thanks for the help, great work...

btw any chance to include this function https://github.com/icefox0801/materialize-autocomplete? just asking... :)

**Comment by mozfet** [comment]
Open a new ticket and tag it as an improvement on https://github.com/mozfet/meteor-autoform-materialize and I will look into it.

**Comment by mozfet** [comment]
It seems my workarounds to remove the modal overlay were no longer needed, some dependency update seems to have fixed it, so my workarounds have been removed, and now the error is gone.
